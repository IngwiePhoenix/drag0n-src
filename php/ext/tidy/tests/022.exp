Warning: tidy_repair_string(): Could not load configuration file '1' in %s on line %d

Warning: tidy_repair_string(): Could not set encoding '1' in %s on line %d

Warning: tidy_repair_string(): Could not load configuration file '' in %s on line %d

Warning: tidy_repair_string(): Could not load configuration file '1' in %s on line %d

Warning: tidy_repair_string(): Could not set encoding '1' in %s on line %d

Warning: tidy_repair_string() expects parameter 1 to be string, array given in %s on line %d

Warning: tidy_repair_file(): Filename cannot be empty in %s on line %d

Warning: tidy_repair_file(): Filename cannot be empty in %s on line %d

Warning: tidy_repair_file() expects parameter 1 to be a valid path, array given in %s on line %d
Done