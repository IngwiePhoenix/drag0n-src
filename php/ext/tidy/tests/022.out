Warning: tidy_repair_string(): Could not load configuration file '1' in /Users/Ingwie/Work/drag0n-php/ext/tidy/tests/022.php on line 7

Warning: tidy_repair_string(): Could not set encoding '1' in /Users/Ingwie/Work/drag0n-php/ext/tidy/tests/022.php on line 7